apps/dcos_dns/src/dcos_dns_app.erl

```diff
@@ -22,6 +22,14 @@
 %%====================================================================
 
 start(_StartType, _StartArgs) ->
+    prometheus_counter:new([{name, dns_forwarder_failures_total},
```

Comment:
- Move metrics to the right module
- export init_metrics to the supervisor
- define the metrics in separate functions